modules/gathering-operator-logs.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting-operator-issues.adoc
+
+[id="gathering-operator-logs_{context}"]
+= Gathering Operator logs
+
+Gather detailed diagnostic information from Operator Pod logs.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Your API service is still functional.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* You have the fully-qualified domain names of the control plane, or master, machines.
+
+.Procedure
+
+. List Operator Pods running in the Operator's namespace, plus Pod status, restarts and age:
```

Comment:
"plus" reads weird to me. Should it be: "... Operator's namespace, including the Pod's status, restarts, and age."

Reply:
Resolved.